packages/gatsby-theme-vtex/src/sdk/search/useSearchFiltersFromPageContext.ts

```diff
@@ -4,6 +4,17 @@ import { useMemo } from 'react'
 import { SearchPageQueryQueryVariables } from '../../templates/__generated__/SearchPageQuery.graphql'
 import { SearchFilterDefaults } from './defaults'
 
+type OrderBy =
+  | 'OrderByScoreDESC'
```

Comment:
We removed this as the default sort in favour of `''`

Reply:
Cool ! done !